src/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react"
+import { Breadcrumb } from "./Breadcrumb"
+import { BreadcrumbItem } from "./BreadcrumbItem"
+
+describe("Breadcrumb", () => {
+  it("renders children", async () => {
+    const breadcrumbLinks = [
+      {
+        href: "#",
+        label: "Home",
+        isCurrent: false,
+      },
+      {
+        href: "#",
+        label: "About",
+        isCurrent: true,
+      },
+    ]
+    const children = <BreadcrumbItem links={breadcrumbLinks} />
+
+    render(<Breadcrumb>{children}</Breadcrumb>)
+
+    expect(screen.getByText("Home")).toBeInTheDocument()
+    expect(screen.getByText("About")).toBeInTheDocument()
+
+    const lastLink = screen.getByText("About")
+    expect(lastLink).toHaveAttribute("aria-current", "page")
```

Comment:
I suggest breaking these out into separate tests. I try to keep 1 expect per test as long as stand-up overhead isn't huge. In the current scenario, if someone removed the `aria-current` attribute the `renders children` test would fail, which isn't very helpful at a glance.

Reply:
@albatrocity thanks for this feedback! I think it would be nice to provide the option to render an `a` or `button` element here. I did consider adding some built in router configuration but didn't want to overcomplicate the component. For example, what if someone consuming the Breadcrumb is using Next.js and needs to render `Link` from `next/link` instead of `@reach/router`?

Reply:
@belv2c yes sorry that I wasn't clear: I'm suggesting being able to pass in a component to render as a Breadcrumb Item as opposed to passing in a `href` prop. That way you could pass in a `Link` from `next/link`. I agree that built-in router config seems out of scope.

I was thinking something like this:

```tsx
import { ReactNode } from "react"
import styles from "./Breadcrumb.module.css"

interface LinkProps {
  href: string
  isCurrent: boolean
  label: string
}

interface Props {
  links: LinkProps[]
  renderItem?: ({
    label,
    index,
    isCurrent,
  }: {
    label: string
    index: number
    isCurrent: boolean
  }) => ReactNode
}

export function BreadcrumbItem({ links, renderItem }: Props) {
  return (
    <>
      {links.map((link, index) =>
        renderItem ? (
          renderItem({ ...link, index })
        ) : (
          <li className={styles.BreadcrumbListItem} key={index}>
            <a
              href={link.href}
              className={styles.link}
              aria-current={link.isCurrent ? "page" : "false"}
            >
              {link.label}
            </a>
            {!link.isCurrent && (
              <span aria-hidden="true" className={styles.Separator}>
                /
              </span>
            )}
          </li>
        ),
      )}
    </>
  )
}

```

That way you could do something like this for custom router support:

```tsx
    const breadcrumbLinks = [
      {
        href: "#",
        label: "Home",
        isCurrent: false,
      },
      {
        href: "#",
        label: "About",
        isCurrent: true,
      },
    ]

return <Breadcrumbs renderItem={({ href, label, isCurrent}) => <Link to={href}>{label}</Link>} />
```